Developing-new-features-with-feature-gating.md

```diff
@@ -145,6 +145,91 @@ spyOnProperty(platformFeatureService, 'status', 'get').and.returnValue(
 );
 ```
 
+#### Backend
+
+For example, if you wish to write a test with the required feature flag enabled, you can do so by following the steps below (the approach to testing with the feature flag disabled is the same up to step 4):
+
+1. Import the following modules into the test file:
+
+```python
+from core.domain import platform_feature_services as feature_services # (alias is optional)
+from core.domain import platform_parameter_domain
+from core.domain import platform_parameter_list
+from core.domain import platform_parameter_registry as registry # (alias is optional)
+```
+
+2. Clear the parameter registry and create a mock feature flag (using the actual enum of the feature flag -- we recommend you do it in the `setUp` hook), like so:
+
+```python
+self.mock_feature_flag = registry.Registry.create_feature_flag(
+  platform_parameter_list.ParamNames.NewFeature, 'a mock of the new_feature feature flag',
+  platform_parameter_domain.FeatureStages.DEV) # FeatureStage DEV just simplifies things
```

Comment:
Can you explain more what you mean by "FeatureStage DEV just simplifies things"? I don't really understand what that's trying to convey.

Reply:
My bad, I meant to imply that I chose (and so should the developer reading it) the DEV stage for the sake of this example (or for the sake of simplicity). I just phrased it weirdly.